src/xrpld/app/tx/detail/NFTokenAcceptOffer.cpp

```diff
@@ -510,4 +525,57 @@ NFTokenAcceptOffer::doApply()
     return tecINTERNAL;
 }
 
+TER
+NFTokenAcceptOffer::checkAcceptAsset(
+    ReadView const& view,
+    ApplyFlags const flags,
+    AccountID const id,
+    beast::Journal const j,
+    Issue const& issue)
+{
+    // Only valid for custom currencies
+    XRPL_ASSERT(
+        !isXRP(issue.currency),
+        "NFTokenAcceptOffer::checkAcceptAsset : input is not XRP");
```

Comment:
```suggestion
        "NFTokenAcceptOffer::checkAcceptAsset : input is XRP");
```

Shouldn't the error message say otherwise?